server/routes/alertsRouter.ts

```diff
@@ -12,20 +11,12 @@ import validationMiddleware from '../middleware/validationMiddleware'
 import { AlertValidator } from '../validators/alertValidator'
 import auditPageAccessAttempt from '../middleware/auditPageAccessAttempt'
 import { Page } from '../services/auditService'
+import { getRequest, postRequest } from './routerUtils'
 
 export default function alertsRouter(services: Services): Router {
   const router = Router()
-
-  const get = (path: string | string[], ...handlers: RequestHandler[]) =>
-    router.get(
-      path,
-      handlers.map(handler => asyncMiddleware(handler)),
-    )
-  const post = (path: string | string[], ...handlers: RequestHandler[]) =>
-    router.post(
-      path,
-      handlers.map(handler => asyncMiddleware(handler)),
-    )
+  const get = getRequest(router)
+  const post = postRequest(router)
```

Comment:
A bit of a tidy up with these get post definitions - if people don't like it then let me know and I'll revert this

Reply:
I think it's a good tidy up, I'm just amazed that we still have to wrap in asyncMiddleware. I'm sure express were talking about removing the need 5 years ago